.github/dependabot.yml

```diff
@@ -137,6 +137,8 @@ updates:
     ignore:
       # breaks compatibility
       - dependency-name: github.com/hashicorp/terraform-plugin-framework
+      # breaks compatibility
+      - dependency-name: github.com/hashicorp/terraform-plugin-go
```

Comment:
How much work does it require for us to support it?

Reply:
I'm not sure exactly what needs to change, but IIRC some changes can be hard to do without changing too much code.
Last time I looked at this was some time ago https://github.com/gravitational/teleport-plugins/issues/684